### DISCUSSION TITLE
怎么调整正文字号为-4/5号以外的字体大小

### DISCUSSION THREAD

**Discussion by ROGERDJQ** [stone-zeng/fduthesis#317] [Q&A] [answered]
您好，感谢提供这么详细的模板以供使用。

在写作中，我想把整体字体调整到-4/5号以外的字体，请问怎么调整？ 比如调整为18pt。

**Comment by ROGERDJQ** [comment]
因为导师要求，我已经设置了页面大小，
`       vmargin    = {2.1 cm,1.3cm},
    inner=2cm,
    outer=1.5cm,
    headheight = 11 pt,`

等于说页面的正文范围已经框定了， 结果要求正文的每一页是26行，因为行距也固定了，这样剩下的只能是把字体调小一些，请问在哪里能新修改这个设置，提前感谢解答和帮助了！

**Comment by stone-zeng** [comment, marked as answer]
```tex
\fontsize{字号}{行距}
\selectfont
```

注意模板自带的 `-4`、`5` 的选项是全局的，还会影响标题、注释等，而上面的方法并不能自动调整。

**Reply by ROGERDJQ** [reply]
最后几个方法结合了一下，这样能够保证脚注的字体和行间距单独定义，正文的字体单独定义。然后把chapter和section的格式又微调了一下。这里分享给后续有需要的人。

P.S. 其实这里有一个问题，\fontsize{字号}{行距}，和\setstretch调整的行距，这两者到底是什么关系，我查了一下setspace包也没有明确说明，然后我自己使用，感觉\fontsize{字号}{行距}里面行距/字号的倍数和\setstretch的倍数也似乎没有关系，也不叠加，也不会倍乘，请问zeng同学知道这两个的关系吗？
```
\renewcommand{\footnotesize}{\setstretch{1.22}\fontsize{8pt}{10pt}\selectfont}
\renewcommand{\normalsize}{\fontsize{10pt}{12pt}\selectfont}
\setcounter{secnumdepth}{0}

\ctexset{
    chapter/format += \zihao{4},
    section/format += \zihao{-4},
  chapter ={
    name = {,},
    number = \chinese{chapter},
    format+=\centering
   },
  section = {
    name   = {,},
    afterskip = 5ex
    % number = \arabic{section}
   },
}

```

**Reply by sikouhjw** [reply]
代码块是三个`而不是一个

**Comment by ROGERDJQ** [comment]
感谢回复！注释我已经手动改了，把脚注的字体和间距都统一做了调整
```tex
\renewcommand{\footnotesize}{\setstretch{1.22}\fontsize{8pt}{10pt}\selectfont}
```
但是正文因为已经有一个字号的设置，我以为可以直接调整字号，就不用全部手动改正文的字体大小了。
这么看可能还是需要手动调整。
再次感谢~
